src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts

```diff
@@ -87,12 +92,18 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit {
    * Currently selected phase split
    */
   selectedPhaseSplit: any = null;
+  showLeaderboardByLatest = false;
```

Comment:
Can you please give a comment for this variable here?

Reply:
changed!